modules/tinymce/src/core/main/ts/keyboard/DeleteBackspaceKeys.ts

```diff
@@ -122,7 +128,7 @@ const setup = (editor: Editor, caret: Cell<Text | null>): void => {
 
   editor.on('keyup', (evt: EditorEvent<KeyboardEvent>) => {
     if (!evt.isDefaultPrevented()) {
-      executeKeyupOverride(editor, evt, isBackspaceKeydown);
+      executeKeyupOverride(editor, evt, isBackspaceKeydown, formatNodes);
     }
```

Comment:
Storing DOM nodes is risky at the best of times. At least clear the array afterwards so there isn't a potential memory leak.


```suggestion
      executeKeyupOverride(editor, evt, isBackspaceKeydown, formatNodes);
      formatNodes.length = 0
    }
```